ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/assertion/defendant/DefendantPartyAssert.java

```diff
@@ -24,119 +24,119 @@ public DefendantPartyAssert(Party party) {
         super(party, DefendantPartyAssert.class);
     }
 
-    public DefendantPartyAssert isEqualTo(CCDDefendant ccdDefendant) {
+    public DefendantPartyAssert isEqualTo(CCDRespondent ccdRespondent) {
         isNotNull();
 
         if (this.actual instanceof Individual) {
-            if (!Objects.equals(INDIVIDUAL, ccdDefendant.getPartyType())) {
+            if (!Objects.equals(INDIVIDUAL, ccdRespondent.getPartyType())) {
```

Comment:
failure messages should also change. It should not be CCDDefendant

Reply:
done